### DISCUSSION TITLE
esp32 wrover t8 v1.7 can not get it to drive ili9341 tft

### DISCUSSION THREAD

**Discussion by jara8462** [Bodmer/TFT_eSPI#1563] [General]
I have the esp32 wrover t8 v1.7 module and trying to get it to drive a ili9341 tft lcd, have been trying by using the examples library  and regardless of how I wire it up the screen stay's white. I have followed posted suggestions from other similiar posts, but no luck. All examples I used compiled with no problems, uploaded to the esp32, but just white screen. I can use putty and other programs and see the esp32 boot and go through the sample programs till finished. Nothing on display, just white screen. I tried a new ili9341 2.4"  SPI  240x320 , and a ili9341 2.8" SPI tft 240x320 v1.2. Both are red boards, I jumpered the across the 5 volt onboard regulator, as both are 5v/3v capable, so the proper voltage level is correct. I get no flickering on the screen regardless of setups. Any suggestions would be appreciated, as I have read through all similiar posts and tried the directions given to them. I have an intel I7 cpu computer, 32 gbytes memory, 3.70 ghz speed, and 11 tbytes of hard drive available. Using arduino v1.8.9 now, all libraries up to date including esp32.

**Comment by Bodmer** [comment]
Run the Read_User_Setup diagnostic example and check the serial output is as expected for pins etc.
